src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs

```diff
@@ -47,13 +96,18 @@
      (react/call :load-data this))
    :load-data
    (fn [{:keys [state]}]
-     (endpoints/call-ajax-orch
-       {:endpoint (endpoints/get-billing-projects)
-        :on-done
-        (fn [{:keys [success? get-parsed-response status-text]}]
-          (if success?
-            (swap! state assoc :projects (get-parsed-response))
-            (swap! state assoc :error-message status-text)))}))})
+     (endpoints/get-billing-projects
+      true
+      (fn [err-text projects]
+        (if err-text
+          (swap! state assoc :error-message err-text)
+          (swap! state assoc :projects projects)))))
+   :-handle-status-change
```

Comment:
I've taken to using this prefix to denote private instance methods. I find it to be a reasonable way to describe my intent.